README.md

```diff
@@ -242,6 +332,13 @@ startup and upon service discovery changes.  `debug` will create very verbose
 log output and should only be used when trying to resolve a problem with the
 agent.
 
+### Windows
+On Windows the agent will log to the console when executed directly in a shell.
```

Comment:
Commas will help readability.

On Windows, the agent will log to the console when executed directly in a shell.
If the agent is configured as a windows service, log events will be logged to the
Windows Event Log.